test/validation/date.validation.spec.ts

```diff
@@ -587,29 +600,4 @@ describe("should chek date functions for custom validation", () => {
   test("should test checkIdentityDate returns true for no param", () => {
     expect(checkIdentityDate()).toBe(true);
   });
-
-  test("should test 1/1/2001 returns true", () => {
```

Comment:
Should this still be returning true or an error? If true, is it covered by another test somewhere?

Reply:
i can add it in with the other true tests

Reply:
updated

Reply:
Thanks.